core/config/uniAssay.js

```diff
@@ -0,0 +1,18 @@
+import harden from '@agoric/harden';
+
+import { makeUniStrategy } from './uniStrategy';
```

Comment:
That's a really good point. I realized that I could take out the custom 'assays' entirely and have the `makeMint` config have a `strategy` slot rather than a `makeAssay` slot. Now there are only the strategies and the customizable `assay.js`